Dockerfile

```diff
@@ -43,6 +43,17 @@ RUN if [ "$TARGETARCH" = "amd64" ]; then \
     fi
 ENV PATH="/root/.risc0/bin:${PATH}"
 
+# Configure Rust for ARM64 QEMU compatibility
+# Use generic CPU target to avoid illegal instructions when running under QEMU emulation
+ARG TARGETARCH
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
+        rustup target add aarch64-unknown-linux-gnu; \
```

Comment:
Why is that necessary? The docker arm image should be built on an arm Ubuntu machine, so that should be installed automatically.

Reply:
Mihir on zeam telegram reported that he was facing this error

`qemu: uncaught target signal 4 (Illegal instruction) - core dumped`